### DISCUSSION TITLE
Bookmarks?

### DISCUSSION THREAD

**Discussion by IBBoard** [CodedOre/NewCaw#36] [Backend]
Twitter have just [announced a bookmarks endpoint](https://twittercommunity.com/t/build-with-bookmarks-on-the-twitter-api-v2/168804).

Twitter Legacy is obviously never going to support it. But could it at least be added for Mastodon and Twitter v2? It [looks like Mastodon supports bookmarks](https://docs.joinmastodon.org/methods/accounts/bookmarks/), but I don't know how much there are forks that add additional features.

**Comment by CodedOre** [comment]
So we're just adding more and more to my list of things we want to add...
But I think if we have the endpoints we could add this.

Backend-wise it would probably a sub-class of `Collection`, frontend-wise I would place it to the lists view, as we already plan to place Likes there.

**Reply by IBBoard** [reply]
Makes sense to me ☺
